<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tlinsWApplayAstroConfiguration</class>
 <widget class="QWidget" name="tlinsWApplayAstroConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QCheckBox" name="chbEnv">
     <property name="text">
      <string>Environment</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>IERS data</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="chbIERSApplay">
        <property name="text">
         <string>Applay</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbIERSOverwrite">
        <property name="text">
         <string>Overwrite</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chnIERSStoreLocaly">
        <property name="text">
         <string>Store localy</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="chbObservationPLace">
     <property name="text">
      <string>Observation place</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
